src/bFS/BFS.java

```diff
@@ -0,0 +1,97 @@
+package bFS;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class BFS {
+	private ArrayList<Node> list;
+	private ArrayDeque<Node> deque;
+	private Node root;
+
+	public BFS() {
+		list = new ArrayList<Node>();
+		deque = new ArrayDeque<Node>();
+		root = null;
+	}
```

Comment:
I think this is very cool that you know how to call a constructor and have an instance of the class in the main method of the class, but this is not necessary. You can simply copy lines 15-16 into the main method without losing anything.